mix.exs

```diff
@@ -1,7 +1,7 @@
 defmodule Membrane.HTTPAdaptiveStream.MixProject do
   use Mix.Project
 
-  @version "0.3.0"
+  @version "0.4.0"
```

Comment:
We are only updating the tests, right? So I don't understand why you bump the version of the package - this test won't even make it into the package

Reply:
Version 0.4.0 is newest master version. When I was developing integration test, @andpodob introduced some changes and 0.4.0 version of one of them. I must have made a mistake when I was resolving conflicts with latest master content and had to bump version manually.